recipes/gnu-time/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gnu-time" %}
```

Comment:
Changed it to `time`. Easier for jinja.

I have no strong opinion on this though.